spec/system/candidate_completing_an_application_spec.rb

```diff
@@ -24,10 +24,25 @@
       click_on t('application_form.save_and_continue')
     end
 
-    it 'successfully submits the application' do
+    it 'can see the success page' do
```

Comment:
I'm not sure about the naming of this spec, I think it's the use of `see`.

Maybe `can show the application submitted page` or `can successfully submit the application` like the previous naming. 🤔